plat/mediatek/mt8183/platform.mk

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+MTK_PLAT      := plat/mediatek
+MTK_PLAT_SOC  := ${MTK_PLAT}/${PLAT}
+
+PLAT_INCLUDES := -I${MTK_PLAT}/common/                            \
+                 -I${MTK_PLAT}/common/drivers/uart/               \
+                 -I${MTK_PLAT_SOC}/include/                       \
+                 -Iinclude/plat/arm/common                        \
```

Comment:
Don't include ``include/plat/arm`` or  ``include/plat/arm/common``. ``drivers/arm`` is fine.